.gitignore

```diff
@@ -38,3 +37,8 @@ docroot/.eslintrc.json
 docroot/.ht.router.php
 docroot/.htaccess
 docroot/INSTALL.txt
+
+# Ignore override docker-compose files.
+# Even those these are in git, we want to allow local changes to them without being pushed up to git.
+docker-compose.override.yml
+prisoner-content-hub-backend-local.env
```

Comment:
This might be a little bit weird to some. But basically I want to add in these files to git, with the default values that they come with.  But can see that developers may want to make additional changes to them, and not have them pushed back to git (or show up as a change in git).  So I've added these files in the PR, but also set them to be ignored.